lang/python27/Portfile

```diff
@@ -64,40 +63,65 @@ depends_lib         port:bzip2 \
 depends_run         port:python_select \
                     port:python2_select
 
-openssl.branch      1.1
+
+if {${subport} eq ${name}} {
+    PortGroup       openssl 1.0
+
+    openssl.branch  1.1
+}
 
 # This port is used by clang-3.4 to bootstrap libcxx
 subport ${name}-bootstrap {
-    set stdprefix ${prefix}
-    prefix ${prefix}/libexec/libcxx-bootstrap
+    revision                [ expr ${revision} + 1 ]
+
+    use_xz                  no
```

Comment:
Unfortunately I don't think this works the way you would expect.

Reply:
I expect that it doesn't try to install `xz` by dependencies for extract phase.

Instead it uses `xz-bootstrap` and provided command to extract an archive.

Also, as it works. Have I missed something?

Reply:
In any port that currently sets `use_xz yes` (autoconf for example), try adding `use_xz no` after that. You'll see that xz is still listed in the extract dependencies.

So this line does nothing, and it's the overwriting of `depends_extract` that removes the xz dep.